sys/include/net/hdlc/fcs.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2016 José Ignacio Alamos <jialamos@uc.cl>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ */
+
+/**
+ * @defgroup    net_hdlc_fcs    FCS calculation
```

Comment:
Maybe this can be unified with [sys_checksum_crc16_ccitt](https://github.com/RIOT-OS/RIOT/blob/master/sys/include/checksum/crc16_ccitt.h) (I know, most likely different polynomial!) somehow?